src/kemal/param_parser.cr

```diff
@@ -41,8 +45,9 @@ module Kemal
     {% end %}
 
     def parse_body
-      return if (@request.headers["Content-Type"]? =~ /#{URL_ENCODED_FORM}/).nil?
-      @body = parse_part(@request.body)
+      return @body = parse_part(@request.body) if !(@request.headers["Content-Type"]? =~ /#{URL_ENCODED_FORM}/).nil?
```

Comment:
Refactoring out `@request.headers["Content-Type"]?` and utilising an early-return guard will clean up this code. (and make it faster as two `headers` accesses aren't needed)